duckduckgo/duckduckgo-privacy-extension
# Fix ABP globals

**Reviewer:** @dharb 



## Description:

**NOTE** This does not merge into develop

This fixes the wonky ABP globals. See notes inline.

## Steps to test this PR:

1. Rebuild the extension (preferably by deleting it and adding it again so it works completely)
2. Visit a few websites - validate easylist lists are loaded correctly (if they didn't load, you'll end up with 0 trackers found)
3. Check a site in the temporary whitelist, it should come up with 0 trackers (e.g. https://open.spotify.com)

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
